data/settings/src/main/java/com/google/jetpackcamera/settings/model/VideoQuality.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.settings.model
+
+import androidx.camera.video.Quality
+
+enum class VideoQuality {
+    DEFAULT,
+    LOWEST,
+    HIGHEST,
+    SD,
+    HD,
+    FHD,
+    UHD;
+
+    companion object {
+        fun toQuality(videoQuality: VideoQuality): Quality? {
```

Comment:
Since this is a conversion to a CameraX class, it should only exist in the core/camera module, not in the model. You can add it as an extension function in `CameraExt.kt` in core/camera:
```
fun VideoQuality.toQuality(): Quality? {
...
}
```